### PR TITLE
Syndicate Fight Club Template

### PR DESCRIPTION
**Pull request by Xymec**
## About The Pull Request
Adds an new template called 'Syndicate Fight Club'

## Why It's Good For The Game
More templates means less antags will suffer from the dreaded writer's block.

## Changelog
:cl:
expansion: Added the Syndicate Fight Club ambition template.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 1]
```diff
@@ -82,3 +82,10 @@
 	job_whitelist = list("Scientist")
 
 //TODO: Everything Past Cook and Scientist Ambitions
+/datum/ambition_template/syndicate_fight_club
+	name = "Syndicate Fight Club"
+	narrative = "Today you're feeling quite pent up and extra hyped. What's an awesome idea to do then? A fight club! It can let you blow off some steam and have some fun!"
+	objective = list("Host a fight club inside of the Holodeck.", "Obtain the Championship Belt.", "Do NOT let anyone shut down your fight club.")
+	tips = list("Not all fight clubs go by the same rules, you can try to get a form of martial arts or CQC in order to get the upperhand.", "A way of obtaining the championship belt is by either, A.) Having a brawl with the captain or a head of staff for it. B.) Breaking into the vault and taking it yourself. C.) Convincing the captain or a head of staff to give it to you ", "Now remember, in fight club there's only one rule, don't talk about fight club. have an uprise against security for trying to stop your fight club, convince the captain to allow it, security and or medical will likely try to shut it down so stay aware of what security and medical are doing. ")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	tips = list("Not all fight clubs go by the same rules, you can try to get a form of martial arts or CQC in order to get the upperhand.", "A way of obtaining the championship belt is by either, A.) Having a brawl with the captain or a head of staff for it. B.) Breaking into the vault and taking it yourself. C.) Convincing the captain or a head of staff to give it to you.", "Now remember, in fight club there's only one rule: Don't talk about fight club. Have an uprise against security for trying to stop your fight club, convince the captain to allow it, security and or medical will likely try to shut it down so stay aware of what security and medical are doing.")
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
I think the objectives could be ever so slightly longer, just so it isn't mega obvious that it's a template objective.

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm (hunk 1 above)
Comment (Xymec) [reply modular_skyrat/modules/ambitions/code/ambition_templates.dm]: Alright. I just updated the objectives to be a bit longer and changed a few things plus I fixed a few mistakes I made for the tips. if there's any other changes needed, feel free to let me know.

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 2]
```diff
@@ -82,3 +82,10 @@
...
+	tips = list("Not all fight clubs go by the same rules, you can try to get a form of martial arts or CQC in order to get the upperhand against your club members and or the station crew.", "A few ways of obtaining the championship belt is by either, A.) Having a brawl with the captain or a head of staff for it. B.) Breaking into the vault and taking it yourself. C.) Convincing the captain or a head of staff to give it to you.", "Now remember, in fight club there's only one rule: Don't talk about fight club. Have an uprise against security for trying to stop your fight club, convince the captain to allow your fight club to exist, security and or medical will likely try to shut it down so stay aware of what security and medical are doing.")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]: I think that would be even clearer, sorry, only had the idea after checking it once again.
```suggestion
	tips = list("Not all fight clubs go by the same rules, you can try to get a form of martial arts or CQC in order to get the upperhand against your club members and or the station crew.", "A few ways of obtaining the championship belt is by either, A.) Having a brawl with the captain or a head of staff for it. B.) Breaking into the vault and taking it yourself. C.) Convincing the captain or a head of staff to give it to you.", "Now remember, in a fight club, there's only one rule: Don't talk about the fight club. Have an uprise against security for trying to stop your fight club, convince the captain to allow your fight club to exist, security and or medical will likely try to shut it down so stay aware of what security and medical are doing.")
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm (hunk 2 above)
Comment (Xymec) [reply modular_skyrat/modules/ambitions/code/ambition_templates.dm]: Don't worry! it's not that big of a deal, anyways it is better for it to be more grammatically correct. :)

Comment (Xymec) [reply modular_skyrat/modules/ambitions/code/ambition_templates.dm]: Should be changed now anyways.

**Review by jjpark-kb — APPROVED** [review]
Awesome to have more people contribute, especially with ambition templates!

File: modular_skyrat/modules/ambitions/code/ambition_templates.dm [hunk 3]
```diff
@@ -82,3 +82,10 @@
 	job_whitelist = list("Scientist")
 
 //TODO: Everything Past Cook and Scientist Ambitions
+/datum/ambition_template/syndicate_fight_club
+	name = "Syndicate Fight Club"
+	narrative = "Today you're feeling quite pent up and extra hyped. What's an awesome idea to do then? A fight club! It can let you blow off some steam and have some fun!"
+	objective = list("This is a really easy thing to do. Legally or illegally host your own fight club for the station crew and try to get a few members to join it.", "Any fight club needs a way to show who's their best fighter! So why don't you go and get the championship belt and brawl with your club members to see who is worthy of owning the belt..", "Prevent ALL acts of shutting down your fight club. The fight club will not end until it's abandoned you unless you have retired from your position and let one of your club members become the new leader.")
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/ambitions/code/ambition_templates.dm]:
```suggestion
	objectives = list("This is a really easy thing to do. Legally or illegally host your own fight club for the station crew and try to get a few members to join it.", "Any fight club needs a way to show who's their best fighter! So why don't you go and get the championship belt and brawl with your club members to see who is worthy of owning the belt..", "Prevent ALL acts of shutting down your fight club. The fight club will not end until it's abandoned you unless you have retired from your position and let one of your club members become the new leader.")
```

**Review by GoldenAlpharex — APPROVED** [review]
I'll fix that for you and we'll get this thing in.

**Pull request merged**